src/Symfony/Component/Cache/Tests/Traits/PdoPruneableTrait.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Tests\Traits;
+
+trait PdoPruneableTrait
+{
+    protected function isPruned($cache, $name)
+    {
+        $o = new \ReflectionObject($cache);
+
+        if (!$o->hasMethod('getConnection') || !$o->hasProperty('table') || !$o->hasProperty('idCol')) {
+            self::fail('Cache does not have "getConnection()" method, or "table" or "idCol" properties.');
+        }
+
+        $getPdoConn = $o->getMethod('getConnection');
+        $getPdoConn->setAccessible(true);
+        $refPropVal = function ($name) use ($cache, $o) {
```

Comment:
tests can know the default value so I would hard code them instead to reduce the boilerplate and logic in test cases

Reply:
Done.